.github/workflows/build.yml

```diff
@@ -142,7 +150,7 @@ jobs:
           file: ImageMagick-${{ steps.variables.outputs.imagemagick_version }}.${{ matrix.target_arch }}.rpm
           classifier: ${{ matrix.nexus_classifier }}
           files: ImageMagick-libs-${{ steps.variables.outputs.imagemagick_version }}.${{ matrix.target_arch }}.rpm
-          classifiers: ${{ matrix.nexus_classifier }}-libs
+          classifiers: libs-${{ matrix.nexus_classifier }}
```

Comment:
this was making insane to manage within ansible:
https://github.com/Alfresco/alfresco-ansible-deployment/pull/892/files#diff-ae5275442eb2794776bfda0cabc6599958cef4a0c8100cec39ef7be7adc9dd29R13

Reply:
we are using the same imagemagick version 7.1.0-16 across all the supported acs versions

the new playbook will requires this newly built packages while the previous playbook versions would still use the previous ones

Reply:
OK so just one new package for each distro. Make sense.